design/volume-policy-label-selector-criteria.md

```diff
@@ -0,0 +1,176 @@
+# Add Label Selector as a criteria for Volume Policy
+
+## Abstract
+Velero’s volume policies currently support several criteria (such as capacity, storage class, and volume source type) to select volumes for backup. This update extends the design by allowing users to specify required labels on the associated PersistentVolumeClaim (PVC) via a simple key/value map. At runtime, Velero looks up the PVC (when a PV has a ClaimRef), extracts its labels, and compares them with the user-specified map. If all key/value pairs match, the volume qualifies for backup.
+
+## Background 
+PersistentVolumes (PVs) in Kubernetes are typically bound to PersistentVolumeClaims (PVCs) that include labels (for example, indicating environment, application, or region). Basing backup policies on these PVC labels enables more precise control over which volumes are processed.
+
+## Goals
+- Allow users to specify a simple key/value mapping (via the field pvcLabels) in the volume policy YAML so that only volumes whose associated PVCs contain those labels are selected.
```

Comment:
So that's only `k=v` labels but not other types of label selectors right?  I think it's good enough but maybe that's worth mentioning in this design.

Reply:
Done, updated non-goals.